cmd/uplink/cmd/root.go

```diff
@@ -76,3 +84,40 @@ func convertError(err error, path fpath.FPath) error {
 
 	return err
 }
+
+func startCPUProf(cmd *cobra.Command, args []string) error {
+	if *debugPprof {
+		f, err := os.Create("cpu.prof")
+		if err != nil {
+			return err
+		}
+		if err := pprof.StartCPUProfile(f); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func stopCPUStartMemProf(cmd *cobra.Command, args []string) error {
+	if *debugPprof {
+		pprof.StopCPUProfile()
+		return runMemoryProf()
+	}
+	return nil
+}
+
+func runMemoryProf() error {
+	f, err := os.Create("memory.prof")
+	if err != nil {
+		return err
+	}
+	runtime.GC()
+	if err := pprof.WriteHeapProfile(f); err != nil {
+		return err
+	}
+	err = f.Close()
```

Comment:
this f.Close can be deferred

Reply:
when i deferred it, i got an error from the linter that i didn't check the err. 
Re: https://build.ops.storj.io/blue/organizations/jenkins/storj/detail/PR-1822/1/pipeline

Reply:
It should be deferred as commented by @mniewrzal because if `pprof.WriteHeapProfile(f)` returns an error the file won't be closed.

Reply:
the file will be closed by the OS once the process exits and since this is the uplink which is a short running process, i think this should be fine to `return f.Close()`